tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/PitclipseMultipleProjectsTest.java

```diff
@@ -66,12 +67,14 @@ public void removeLaunchConfigurations() throws CoreException {
     public void testExecutionMode() throws CoreException {
         try {
             PitPreferenceSelector selector = PAGES.getWindowsMenu().openPreferences().andThen();
+            bot.waitUntil(new DurationElapsed(200));
```

Comment:
This should not be needed (see also above): SWTbot has its property for such things and we should not rely on these forced sleeps. This https://github.com/pitest/pitclipse/pull/228 could help for flaky tests, if that's the reason.

Reply:
It was a quick workaround to prevent some tests from failing/freezing locally. I noticed afterwards that failures were actually due to the tests being run in the UI thread. I'll clean up.